src/scripts/destinyTrackerApi/reviewSubmitter.js

```diff
@@ -0,0 +1,70 @@
+import { itemTransformer } from './itemTransformer.js';
+
+class reviewSubmitter {
+  constructor($q, $http, dimPlatformService, trackerErrorHandler, loadingTracker, scoreMaintainer) {
+    this.$q = $q;
+    this.$http = $http;
+    this._itemTransformer = new itemTransformer();
+    this._trackerErrorHandler = trackerErrorHandler;
+    this._dimPlatformService = dimPlatformService;
+    this._loadingTracker = loadingTracker;
+    this._reviewDataCache = scoreMaintainer;
+  }
+
+  getReviewer() {
+    var membershipInfo = this._dimPlatformService.getActive();
```

Comment:
I hadn't considered that possibility.  It will fail, but DTR's API needs membership data along with the item being reviewed, so failing is expected.

There's no indication in my revision of the UI that it failed or succeeded, so, uh.